src/bot/plugins/admin.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+from dependency_injector.wiring import Provide, inject
+from mmpy_bot import Message, Plugin, listen_to
+
+from src.bot.services.admin import AdminService
+from src.core.db.models import Admin
+from src.depends import Container
+
+
+class BotAdmin(Plugin):
+    @inject
+    @listen_to("Admin", re.IGNORECASE)
+    async def admin(self, message: Message, admin_service: AdminService = Provide[Container.admin_service]):
```

Comment:
Не там стоит `inject` - должен быть обязательно первым.  
```suggestion
    @listen_to("Admin", re.IGNORECASE)
    @inject
    async def admin(self, message: Message, admin_service: AdminService = Provide[Container.admin_service]):
```